### DISCUSSION TITLE
How do I add a umask so that uploaded files have the correct permissions? (unraid)

### DISCUSSION THREAD

**Discussion by StudioNirin** [drakkan/sftpgo#1582] [Q&A]
I have tried adding in an extra parameter '--UMASK 777' but that didn't have any effect, I then added a new variable field with the UMASK key and the value of 777, but that also had no effect. 

Uploaded files by default have the owner 'nobody' and the permissions of -rw--r--r-  when I need for them to have rwrwrw. 

I know there's a umask line in the sftpgo.json file, at least according to the documentation, but there's nothing that tells me how to actually access the edit that file in docker.

**Comment by StudioNirin** [comment]
Anyone got any ideas on this?

**Comment by faevourite** [comment]
Set the `SFTPGO_COMMON__UMASK` envar.
